Source/WebKit/WPEPlatform/wpe/WPEGestureDetector.cpp

```diff
@@ -49,6 +49,10 @@ void GestureDetector::handleEvent(WPEEvent* event)
         break;
     case WPE_EVENT_TOUCH_MOVE:
         if (double x, y; wpe_event_get_position(event, &x, &y) && m_position) {
+            if (!wpeSettings)
+                wpeSettings = adoptGRef(WPE_SETTINGS(g_object_new(WPE_TYPE_SETTINGS, nullptr)));
```

Comment:
We shouldn't create the settings here. You can get the display settings from the WPEEvent with `wpe_display_get_settings(wpe_view_get_display(wpe_event_get_view(event)));`